homeassistant/components/prometheus/__init__.py

```diff
@@ -90,7 +92,7 @@
 
 def setup(hass: HomeAssistant, config: ConfigType) -> bool:
     """Activate Prometheus component."""
-    hass.http.register_view(PrometheusView(prometheus_client))
+    hass.http.register_view(PrometheusView(prometheus_client,config))
```

Comment:
Whitespace missing...
It seems that you skipped the pre-commit hooks :(

Reply:
done